### PR TITLE
kevin/ENG-994

### PR DESCRIPTION
**Pull request by kcvan**
## Description
Why did you write this code?
start(work-login): start work login
## How to Test
- [ ] Visit /work/login
- [ ] login with email/password
- [ ] verify that it brings you to the /work page
- [ ] verify that signing in with facebook and google also bring you to the /work page

*note

you'll have to go back to the `/` page to sign out after each login

### CODE REVIEW

File: src/pages/login/LoginTestimonials.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { Col } from 'reactstrap';
+
+const LoginTentimonials = () => (
+  <Col lg="5" xl="4" className="d-none d-lg-flex align-items-center gradient-half-primary-v1 height-lg-100vh px-0">
+
+  </Col>
+);
+
+export default LoginTentimonials;
```
Comment (woeltjen) [inline src/pages/login/LoginTestimonials.tsx]: Tentimonials 🐙

Comment (kcvan) [reply src/pages/login/LoginTestimonials.tsx]: yall never heard of tentimonials?

Comment (kcvan) [reply src/pages/login/LoginTestimonials.tsx]: living under a rock

**Comment by kcvan** [comment]
@woeltjen @tc @jerminatorhits would probably be good to move these cdns into a npm module per vic's suggestion

**Pull request merged**